test/pages/uat_actions/generate_test_run.spec.js

```diff
@@ -75,7 +75,7 @@ describe('UAT Actions', () => {
     })
 
     it('when busy', () => {
-      wrapper.vm.$data.status = statuses.Busy
+      wrapper.vm.$data.status = Status.Busy
```

Comment:
`wrapper.vm` could be refactored throughout, and if `status` is a `data`  attribute for the component, maybe it doesn't need to be accessed via `$data`? Not 100% sure though

Reply:
I think I learned recently that if you set values using `.data` instead of `.$data` the reactive parts of Vue are not triggered.  `.data` is a read-only view of the VM's data values.